### ISSUE TITLE
Publish skips Our.Umbraco.AuthU.dll

### ISSUE THREAD

**Issue by rmagar** [mattbrailsford/umbraco-authu#50]
I installed version 1.0.3 via Nuget on a Umbraco 7.7.1 web project. Everything works great when I try my website locally. However, when I try to Publish (Web Deploy) the website hosted on a VM, the main dll (Our.Umbraco.AuthU.dll) does not get copied by Publish. I am on Visual Studio 2017. I won't be able to try version 2 since we won't be moving to Umbraco 8 any time soon. Could you help resolve the issue with Publish? Thanks.

**Comment by mattbrailsford** [comment]
I’m not that familiar with cloud, but at a guess, I think you may not have the dll committed to your git repository. With cloud I believe all your bin folder should be committed to the repo.

**Comment by rmagar** [comment]
No, I tried publishing to a staging site on a VM in the same LAN as my machine. I noticed that Our.Umbraco.AuthU.dll is not strong name signed but all the other dependent dlls are. Could that be the issue? My own attempt to strong name sign it didn't help.
